src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const longTerm = 7;
+  const price = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < shortTerm) {
+    return price * days;
+  }
+
+  if (days >= shortTerm && days < longTerm) {
+    return (price * days) - discountSmall;
+  }
+
+  if (days >= longTerm) {
+    return (price * days) - discountBig;
+  }
```

Comment:
use early return pattern
```suggestion
    return (price * days) - discountBig;
```